pages/resources.tsx

```diff
@@ -155,4 +104,10 @@ function ResourcePanel({
   );
 }
 
+export const getStaticProps: GetStaticProps = async () => {
+  const resourcesAndContactsContent = await fetchResources();
+
+  return { props: { resourcesAndContactsContent } };
```

Comment:
Each of the pages need the `revalidate` argument. This is what allows the page to be rebuilt with new changes from Contentful after this PR is deployed. Example from the landing page:

```typescript
  return {
    props: { landingContent },
    // Next.js will attempt to re-generate the page:
    // - When a request comes in
    // - At most once every 60 second
    revalidate: 60,
  };
```